### PR TITLE
Update cursor-directory extension

### PR DESCRIPTION
**Pull request by escwxyz**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/cursor-directory/src/index.tsx [hunk 1]
```diff
@@ -66,111 +74,108 @@ export default function Command() {
...
+                  actions={
+                    <ActionPanel>
+                      <ActionPanel.Section title="Actions">
+                        <ViewRuleAction cursorRule={cursorRule} popularOnly={popularOnly} />
+                        <CopyRuleAction cursorRule={cursorRule} />
+                        <ToggleViewAction showingDetail={showingDetail} setShowingDetail={setShowingDetail} />
+                        <Action
+                          title="Unstar Cursor Rule"
+                          icon={Icon.StarDisabled}
+                          shortcut={{ modifiers: ["cmd"], key: "u" }}
```
Comment (pernielsentikaer) [inline extensions/cursor-directory/src/index.tsx:116]: In Raycast we use the same hotkey for both unpin/pin - you can use `Keyboard.Shortcut.Common.Pin`

File: extensions/cursor-directory/src/index.tsx [hunk 2]
```diff
@@ -66,111 +74,108 @@ export default function Command() {
...
+                        <ViewRuleAction cursorRule={cursorRule} popularOnly={popularOnly} />
+                        <CopyRuleAction cursorRule={cursorRule} />
+                        <ToggleViewAction showingDetail={showingDetail} setShowingDetail={setShowingDetail} />
                         <Action
-                          title="Copy Cursor Rule"
-                          icon={Icon.Clipboard}
-                          shortcut={{ modifiers: ["cmd"], key: "c" }}
+                          title="Star Cursor Rule"
+                          icon={Icon.Star}
+                          shortcut={{ modifiers: ["cmd"], key: "s" }}
```
Comment (pernielsentikaer) [inline extensions/cursor-directory/src/index.tsx:169]: In Raycast we use the same hotkey for both unpin/pin - you can use `Keyboard.Shortcut.Common.Pin`

File: extensions/cursor-directory/src/index.tsx (hunk 1 above)
Comment (escwxyz) [reply extensions/cursor-directory/src/index.tsx:116]: Thanks for pointing!

**Comment by pernielsentikaer** [comment]
You can import `Keyboard` from `@raycast/api` so you can use it as I did to ensure it's always the same 🙂

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**